docs/ipycytoscape_example.rst

```diff
@@ -0,0 +1,137 @@
+====================
+IPyCytoscape Example
+====================
+
+Thebe can display output from ipycytoscape_, which could visualize graphs using Cytoscape.js_.

```

Comment:
Mention the license of the code you copied from ipycytoscape (if you did) like we did in the other examples.

Reply:
Added the mention to the BSD-3 Clause license.